File: ClientApp/src/models/FixtureDto.ts
import { EntryDto } from "./EntryDto";
import { EventDto } from "./EventDto";

export interface FixtureDto {
	id: number;
	name: string;
	events?: EventDto[];
	entries?: EntryDto[];
	deadline?: string;
	locked: boolean;
	tiebreakerEventId: string;
	isComplete: boolean;
}

File: Services/Wingrid.Services.EventAPI/Models/Dto/FixtureDto.cs
namespace Wingrid.Services.EventAPI.Models.Dto
{
    public class FixtureDto
    {
        public int Id { get; set; }
        public required string Name { get; set; }
        public List<EventDto> Events { get; set; } = [];
        public List<Entry> Entries { get; set; } = [];
        public DateTime Deadline { get; set; }
        public bool Locked { get; set; }
        public required string TiebreakerEventId { get; set; }
        public bool IsComplete { get; set; }
    }
}